postman/camp-trello-clone.postman_attachments.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
add this collection as a folder to existing collection. Don't create another one